frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "filepond-plugin-file-validate-type": "^1.2.1",
     "filepond-polyfill": "^1.0.2",
     "hoist-non-react-statics": "^3.2.1",
+    "js-file-download": "^0.4.4",
```

Comment:
Did you get a chance to check that this new dependency works with IE?

Reply:
Not yet, this is a good point. Will add this note to PR#490 as we merge them together; PR#490 also has a new js library to test with IE.